### PR TITLE
Adding a Baz tile

### PR DESCRIPTION
**Pull request by nimrodkor**
### What does this PR do?

This PR adds a tile for [Baz](https://baz.co)

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors.

### CODE REVIEW

**Comment by cswatt** [comment]
Added DOCS-8694 to review.

File: baz/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Once integrated, you will see the integration with its status in the [Datadog integration page][2].
+
+### Uninstallation
+
+Visit the [Datadog integration page][2] and click the "Remove" trash button to uninstall the integration.
+
+## Data Collected
+
+Baz collects APM metrics - traces, spans and services, to get the app context so missing in today's code reading and reviewing.
```
Comment (maycmlee) [inline baz/README.md:25]:
```suggestion
Baz collects APM metrics on traces, spans and services, to provide information on your apps during code reviews.
```

File: baz/manifest.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "4bacfa32-13d5-4e4d-9339-48376f7999df",
+  "app_id": "baz",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Connect your code with tracing data, bringing the context of the application into your codebase",
```
Comment (maycmlee) [inline baz/manifest.json:11]:
```suggestion
    "description": "Connect your code with tracing data to bring in information about your the application into your codebase",
```

File: baz/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Baz
+
+## Overview
+Baz helps developers write unbreakable code by:
```
Comment (maycmlee) [inline baz/README.md:4]: I'd suggest changing `unbreakable` to `thoroughly validated`.

**Review by maycmlee — COMMENTED** [review]
Left some feedback!

File: baz/README.md (hunk 3 above)
Comment (nimrodkor) [reply baz/README.md:4]: It's the brand... https://baz.co

**Review by maycmlee — APPROVED** [review]
Approving for docs, but there is an error in the manifest.json that needs to be fixed. Engineering should be able to provide more guidance on it.

**Comment by emarsha94** [comment]
Per Dominics email, closing the PR

**Pull request closed without merging**